### PR TITLE
done --auth

### PR DESCRIPTION
**Pull request by itsmebipul00**
In this PR I have added single auth with signIn/singUp

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -3,33 +3,33 @@ import { createGlobalStyle } from 'styled-components'
...
-    --letter-spacing-3: 2px;
-
-    --fw-300: 300;
+		--fs-900: 3rem;
+		--fs-800: 2rem;
+		--fs-700: 1.75rem;
+		--fs-600: 1.5rem;
+		--fs-500: 1.25rem;
+		--fs-400: 1rem;
+		--fs-300: 0.75rem;
```
Comment (bossmonkey98) [inline src/GlobalStyles.js:12]: use px here

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -3,33 +3,33 @@ import { createGlobalStyle } from 'styled-components'
...
+		--fs-200: 0.5rem;
```
Comment (bossmonkey98) [inline src/GlobalStyles.js:13]: here

File: src/Pages/SignInPage/styles/Form.styled.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		width: 20rem;
+	}
+
+	& > *:not(:last-child) {
+		margin-bottom: 1.25rem;
+	}
+
+	input {
+		width: 100%;
+		padding: 0.5rem;
```
Comment (bossmonkey98) [inline src/Pages/SignInPage/styles/Form.styled.js:21]: use px here

File: src/Pages/SignInPage/styles/Form.styled.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+	label[for='remember'] {
+		font-size: var(--fs-400);
+	}
+
+	input[type='checkbox'] {
+		display: inline;
+		width: 1rem;
+		height: 1rem;
+		margin-right: 0.5rem;
```
Comment (bossmonkey98) [inline src/Pages/SignInPage/styles/Form.styled.js:32]: here

File: src/Pages/SignInPage/styles/Form.styled.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+		cursor: pointer;
+	}
+
+	.span-flex {
+		display: flex;
+	}
+
+	button {
+		padding: 0.25rem 0.75rem;
```
Comment (bossmonkey98) [inline src/Pages/SignInPage/styles/Form.styled.js:41]: here

**Review by bossmonkey98 — APPROVED** [review]
good to merge

**Review by sudipkundu999 — APPROVED** [review]
The UI looks so clean in dark mode!
Appreciation:  Good to see you're diversifying your CSS skills.
Good to merge

**Pull request merged**